src/util/events.js

```diff
@@ -57,10 +58,8 @@ export function computeFilteredEvents(events, filters, zipcodes) {
     }
 
     if (filters.zipcode) {
-      const zipcodeLatLng = L.latLng(zipcodes[filters.zipcode]);
-
       const milesFromZipcode = metersToMiles(
-         zipcodeLatLng.distanceTo([event.lat, event.lng])
+        distance([zipcodes[filters.zipcode][1], zipcodes[filters.zipcode][0]], [event.lng, event.lat])
```

Comment:
Note: we are generating the valid zipcodes file, we could reorg the coords order to be `[lng, lat]` if that is more useful/in line with mapbox-gl , seems like this would make a couple things in here slightly simpler...LMK if you think it is worth it, happy to make that change!